changelog/unreleased/features/6610.md

```diff
@@ -0,0 +1,18 @@
+- Added `RouteRefreshController` interface to manage route refreshes. Retrieve it via `MapboxNavigation#routeRefreshController`.
```

Comment:
Does exposing `RouteRefreshController` makes our SDK easier to harder to change?
I found myself in a complex situations with reroute logic a few times. It was very hard to change its structure, because we exposed it to the users in our API. Maybe exposing `MapboxNavigation#refreshRoute()` leaves us more flexibility for future changes? 🤔

Reply:
It's just an interface that accumulates refresh-related API.